guides/Other Subjects/PieceStores.md

```diff
@@ -158,38 +86,56 @@ This is great and all, but we need to register these pieces/store:
 
 ```javascript
 const { Client } = require('klasa');
-const GenreStore = require('./GenreStore');
+const RawEventStore = require('./RawEventStore');
 
-class MySwankyMusicBot extends Client {
+class MyBot extends Client {
 
 	constructor(...args) {
 		super(...args);
 
-		// Make a new GenreStore and attach it to client
-		this.genres = new GenreStore();
+		// Make a new RawEventStore and attach it to client
+		this.rawEvents = new RawEventStore();
 
-		// Register the GenreStore to be loaded, init, and available
+		// Register the RawEventStore to be loaded, init, and available
 		// to be used as an arg to be looked up in commands
-		this.registerStore(this.genres);
-
-		// Note: You will likely want to make a custom Argument for this new piece type
+		this.registerStore(this.rawEvents);
 	}
 
 }
 
-new MySwankyMusicBot().login('token-goes-here');
+new MyBot().login('token-goes-here');
 ```
 
-Then we use it in our player class I completely made up earlier like so:
+Now to actually make our raw events work! This code goes in events/raw.
 
 ```javascript
-async () => {
-	// Without going over setting up guild configs,
-	// or actually writing a player class.
-	const nextSong = await this.client.genres.get(this.guild.configs.genre)
-		.getNext(this);
-
-	// nextSong should now be a pseudo random song based on the
-	// genre seeds and what has recently played.
+const { Event } = require('klasa');
+const { Constants: { Events } } = require('discord.js');
+
+module.exports = class extends Event {
+
+	async run(data) {
+		const event = Events[data.t];
+		if (this.client.rawEvents.has(event)) {
+			const rawEvent = this.client.rawEvents.get(event);
+			if (rawEvent.enabled) rawEvent.run(data.d);
+		}
+	}
+
+};
+```
+
+Now an example raw event, we'll have messageReactionAdd, which will go in rawEvents/messageReactionAdd
```

Comment:
`Now, for an example event, we'll have messageReactionAdd, which will go in rawEvents/messageRreactionAdd and will run on all added reactions` or close to this.